fix compiler (gcc 4.8) errors for ruby-gnome2:

rbgtkcalendar.c: In function 'rg_display_options':
/usr/include/ruby-2.0/ruby/ruby.h:446:18: error: expected '(' before '!' token
 #define NIL_P(v) !((VALUE)(v) != Qnil)

seems same error as here:
https://github.com/shoes/shoes/issues/246
--- ruby-gnome2-all-1.1.9/gtk2/ext/gtk2/rbgtkcalendar.c~	2012-01-05 13:23:51.000000000 +0200
+++ ruby-gnome2-all-1.1.9/gtk2/ext/gtk2/rbgtkcalendar.c	2014-07-26 22:30:31.050939929 +0300
@@ -105,7 +105,7 @@
     VALUE flags;
     rb_scan_args(argc, argv, "01", &flags);
 
-    if NIL_P(flags){
+    if (NIL_P(flags)){
 #if GTK_CHECK_VERSION(2,4,0)
         return GFLAGS2RVAL(gtk_calendar_get_display_options(_SELF(self)),
                            GTK_TYPE_CALENDAR_DISPLAY_OPTIONS);
--- ruby-gnome2-all-1.1.9/gtk2/ext/gtk2/rbgdkscreen.c~	2012-01-05 13:23:51.000000000 +0200
+++ ruby-gnome2-all-1.1.9/gtk2/ext/gtk2/rbgdkscreen.c	2014-07-26 22:30:54.818646494 +0300
@@ -210,7 +210,7 @@
     VALUE value;
 
     rb_scan_args(argc, argv, "11", &name, &type);
-    if NIL_P(type) 
+    if (NIL_P(type))
         gtype = G_TYPE_STRING;
     else
         gtype = CLASS2GTYPE(type);
--- ruby-gnome2-all-1.1.9/gtk2/ext/gtk2/rbgdkevent.c~	2012-01-05 13:23:51.000000000 +0200
+++ ruby-gnome2-all-1.1.9/gtk2/ext/gtk2/rbgdkevent.c	2014-07-26 22:31:06.012469585 +0300
@@ -387,7 +387,7 @@
     VALUE value;
 
     rb_scan_args(argc, argv, "11", &name, &type);
-    if NIL_P(type) 
+    if (NIL_P(type))
         gtype = G_TYPE_STRING;
     else
         gtype = CLASS2GTYPE(type);
--- ruby-gnome2-all-1.1.9/gdk3/ext/gdk3/rbgdkscreen.c~	2012-01-05 13:23:51.000000000 +0200
+++ ruby-gnome2-all-1.1.9/gdk3/ext/gdk3/rbgdkscreen.c	2014-07-26 22:31:15.812898377 +0300
@@ -214,7 +214,7 @@
     VALUE value;
 
     rb_scan_args(argc, argv, "11", &name, &type);
-    if NIL_P(type) 
+    if (NIL_P(type))
         gtype = G_TYPE_STRING;
     else
         gtype = CLASS2GTYPE(type);
--- ruby-gnome2-all-1.1.9/gdk3/ext/gdk3/rbgdkevent.c~	2012-01-05 13:23:51.000000000 +0200
+++ ruby-gnome2-all-1.1.9/gdk3/ext/gdk3/rbgdkevent.c	2014-07-26 22:31:26.213353421 +0300
@@ -370,7 +370,7 @@
     VALUE value;
 
     rb_scan_args(argc, argv, "11", &name, &type);
-    if NIL_P(type) 
+    if (NIL_P(type))
         gtype = G_TYPE_STRING;
     else
         gtype = CLASS2GTYPE(type);
